Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_006.GitHubPat.ps1

```diff
@@ -5,11 +5,11 @@
 
 # AppSettings style
 <appSettings>
-  <add key="GitHubPat" value="ff34885a8624460a855540c6592698d2f1812843" />
+  <add key="GitHubPat" value="dead885a8624460a855540c6592698d2f1812843" />
```

Comment:
>dead885a8624460a855540c6592698d2f1812843 [](start = 30, length = 40)

Note that due to this change when you create test cases, each case must produce a unique fingerprint, or some results will be filtered away. :) For all test changes, I've tweaked some portion of the key/secret to make sure each case produces a unique fingerprint.